source/kraken/maintenance_worker.cpp

```diff
@@ -366,6 +366,7 @@ void MaintenanceWorker::handle_rt_in_batch(const std::vector<AmqpClient::Envelop
     bool autocomplete_rebuilding_activated = false;
     auto rt_action = RTAction::chaos;
 
+    size_t applied_entity_number = 0u;
```

Comment:
I'm often disturbed by names ending with `number`. Does it mean the identifier (the number) of an applied entity, or is it the counting of applied entities? In this case, I believe it's the count so I would suggest either `applied_entity_count` (or `number_of_applied_entities`?).

Reply:
Switeched all to `count` in https://github.com/hove-io/navitia/pull/3955/commits/e7122af1bdb0aaec2c912e7f5dde1bb3f60b9c68